docs/_templates/overrides/metpy.calc.rst

```diff
@@ -174,6 +174,7 @@ calc
       reduce_point_density
       resample_nn_1d
       smooth_gaussian
+      smooth_n_pt
```

Comment:
Need to update this to `smooth_n_point`.

Reply:
Got it.